src/main/java/net/openhft/chronicle/wire/WireType.java

```diff
@@ -600,47 +599,6 @@ public boolean tryAdvance(@NotNull Consumer<? super T> action) {
                 }, false);
     }
 
-    @NotNull
-    public <T> Map<String, T> fromFileAsMap(String filename, @NotNull Class<T> tClass) throws IOException, InvalidMarshallableException {
```

Comment:
should deprecate before removing

Reply:
This method is not used anywhere (across all Chronicle repos) but in the following removed test. 
IMO it does not make sense to spend more time to fix it (I've tried) as it's a dead code.

Reply:
OK. @peter-lawrey 's decision.... I am OK to remove although if we want to preserve API compatibility we could deprecate but replace the contents with `throw new UnsupportedOperationException()`

Reply:
I wouldn't throw UOE; it's better to fail at compile time.

Reply:
so @peter-lawrey you would rather remove the methods?

Reply:
All to be deprecated for removal in x.27